extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -1110,12 +1127,12 @@ static public function substVariables( $cv_value, $city_id ) {
 	 */
 	static public function DBtoID( $city_dbname, $master = false ) {
 
-		if ( ! self::isUsed() ) {
+		if ( ! static::isUsed() ) {
 			Wikia::log( __METHOD__, "", "WikiFactory is not used." );
 			return false;
 		}
 
-		$oRow = self::getWikiByDB( $city_dbname, $master );
+		$oRow = static::getWikiByDB( $city_dbname, $master );
 
 		return isset( $oRow->city_id ) ? $oRow->city_id : false;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1125)